src/userfunc.c

```diff
@@ -172,6 +174,7 @@ one_function_arg(
 		type = vim_strsave((char_u *)
 					    (is_vararg ? "list<any>" : "any"));
 	    ((char_u **)argtypes->ga_data)[argtypes->ga_len++] = type;
+	    ((int8_T *)is_argobjm->ga_data)[is_argobjm->ga_len++] = 0;
```

Comment:
Can you rename "is_argobjm" to "is_arg_objm" or "arg_objm" (it is more clear)?

Reply:
I'll rename to `arg_objm`.